src/app/helptext/services/components/service-ups.ts

```diff
@@ -41,7 +41,14 @@ ups_driver_tooltip : T('See the <a\
 ups_driver_validation : [ Validators.required ],
 
 ups_port_placeholder : T('Port'),
-ups_port_tooltip : T('Enter the serial or USB port the UPS is plugged into.'),
+ups_port_tooltip : T('Enter the serial or USB port the UPS is \
+ connected to.<br /> <br />\
```

Comment:
```suggestion
 using.<br /> <br />\
```
Trying to avoid ending the sentence with a preposition.